pkg/platform/runtime/store/store.go

```diff
@@ -331,3 +335,20 @@ func (s *Store) StoreBuildExpression(expr *buildexpression.BuildExpression, comm
 	}
 	return fileutils.WriteFile(s.buildExpressionFile(), data)
 }
+
+var ErrNoBuildScriptFile = errs.New("no buildscript file")
+
+func (s *Store) BuildScript() (*buildscript.Script, error) {
+	if !fileutils.FileExists(s.buildScriptFile()) {
+		return nil, ErrNoBuildScriptFile
+	}
+	bytes, err := fileutils.ReadFile(s.buildScriptFile())
+	if err != nil {
+		return nil, errs.Wrap(err, "Could not read buildscript file")
+	}
+	return buildscript.New(bytes)
+}
+
+func (s *Store) StoreBuildScript(script *buildscript.Script) error {
```

Comment:
I'm not finding a reference to this being called.

Reply:
Thanks for catching this oversight on my part.